charts/umbrella/values.yaml

```diff
@@ -539,8 +539,31 @@ sdfactory:
     # -- Details for Clearing House Client Secret
     clearingHouseClientSecret: ""
 
+# -- Map of company BPN and company name. For each of these Companies, a MIW wallet will be created.
+companies:
```

Comment:
Might make sense to move this map to Values.managed-identity-wallet.seeding, as it belongs to the miw seeding

Reply:
Good point. Adjusted it